web/js/map/wv.map.rotate.js

```diff
@@ -146,15 +144,28 @@ wv.map.rotate = wv.map.rotate || function(ui, models, map) {
    * @returns {void}
    */
   this.updateRotation = function() {
-    var deg, radians, view;
+    var deg,
+      radians,
+      view,
+      currentDeg;
 
     view = ui.selected.getView();
     radians = view.getRotation();
     models.map.rotation = radians;
     self.setResetButton(radians);
-  };
-
 
+    currentDeg = (view.getRotation() * (180.0 / Math.PI));
```

Comment:
@ZachTRice this logic looks identical to what's happening in `this.rotate` . Should probable isolate that into it's own method and use it in both places